src/msw/webview_ie.cpp

```diff
@@ -338,33 +373,40 @@ wxWebViewZoom wxWebViewIE::GetIETextZoom() const
 
 void wxWebViewIE::SetIEOpticalZoom(wxWebViewZoom level)
 {
-    //We do not use OLECMDID_OPTICAL_GETZOOMRANGE as the docs say the range
-    //is 10 to 1000 so the check is unnecessary
-    VARIANT zoomVariant;
-    VariantInit (&zoomVariant);
-    V_VT(&zoomVariant) = VT_I4;
+    float zoom;
```

Comment:
Hmm, don't you get warnings about this possibly being uninitialized? I think we need to set it to something (0?) in `default:` below.